iot-hub/Quickstarts/device-streams-proxy/device/Program.cs

```diff
@@ -31,8 +31,8 @@ public static class Program
         private static string s_port = Environment.GetEnvironmentVariable("REMOTE_PORT");
 
         // Select one of the following transports used by DeviceClient to connect to IoT Hub.
-        private static TransportType s_transportType = TransportType.Amqp;
-        //private static TransportType s_transportType = TransportType.Mqtt;
+        //private static TransportType s_transportType = TransportType.Amqp;
```

Comment:
Need to confirm if C# SDK team has a standard of having AMQP as the default option.
Are you changing this on purpose?

Reply:
If anything, I prefer using MQTT as default. MQTT is the protocol we have repeatedly heard from customers to have more support for.

Reply:
Done